vm/systemSmartContracts/staking.go

```diff
@@ -1447,7 +1447,7 @@ func (s *stakingSC) getOwner(args *vmcommon.ContractCallInput) vmcommon.ReturnCo
 		return vmcommon.UserError
 	}
 
-	s.eei.Finish([]byte(hex.EncodeToString(stakedData.OwnerAddress)))
+	s.eei.Finish(stakedData.OwnerAddress)
```

Comment:
There still are some s.eei.Finish() with hexEncode on the RewardAddress: lines 1273 and 1384. Should not all be not hexEncoded?

Reply:
I have implemented this and indeed I have used the other examples in the system SC. What we have figure out today is that this hex encoding is useless as we need a complementary hex.Decode call when processing internal data. The code is now slightly optimized and one bug was fixed. If we need an API function for the owner field we will create a new function that will return the data in hex format.